ocs/ocp.py

```diff
@@ -257,11 +257,9 @@ def get_ceph_tools_pod():
         str: The Ceph tools pod name
     """
     ocp_pod_obj = OCP(kind='pods', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
-    ct_pod = ocp_pod_obj.get(
-        selector='app=rook-ceph-tools'
-    ).toDict()['items'][0]['metadata']['name']
-    assert ct_pod, f"No Ceph tools pod found"
-    return ct_pod
+    ct_pods = ocp_pod_obj.get(selector='app=rook-ceph-tools')
+    assert ct_pods, f"No Ceph tools pods found"
```

Comment:
Instead can we raise an exception, so that user can retry?

Reply:
@vasukulkarni not sure I follow. It does raise an exception, an assertion error

Reply:
@ebenahar assert indicates a strong failure, we can just raise exception "CephToolsPodNotReady" and it will be clear at try/except block.

Reply:
If we are concerned with wanting to retry can't we use `oc wait --for condition=ready` instead of `get`? This way we can configure the appropriate timeout and still just simply assert the success of that command?